packages/backend/src/app.js

```diff
@@ -0,0 +1,64 @@
+import './env'
```

Comment:
This diff is misleading. I moved the old index.js to app.js and moved the code into two functions. The real diff is something like:
```diff
+export const startApp = () => {
...
-  if (__DEV__) console.log(chalk.yellow('> in development'))
+  if (process.env.NODE_ENV === 'development') console.log(chalk.yellow('> in development'))
...
+  return app;
+};

+export const connectToDb = () => {
...
+};
```